cointop/table_header.go

```diff
@@ -123,6 +128,15 @@ var HeaderColumns = map[string]*HeaderColumn{
 	},
 }
 
+// GetLabel fetch the label to use for the heading (depends on configuration)
+func (ct *Cointop) GetLabel(h HeaderColumn) string {
```

Comment:
I think Go best practice would be for `HeaderColumn` to be passed by reference here

Reply:
Fixed